Specs/Scene/ModelSpec.js

```diff
@@ -47,6 +47,7 @@ defineSuite([
     var texturedBoxSeparateUrl = './Data/Models/Box-Textured-Separate/CesiumTexturedBoxTest.gltf';
     var texturedBoxCustomUrl = './Data/Models/Box-Textured-Custom/CesiumTexturedBoxTest.gltf';
```

Comment:
Just a heads up that this model needs to be updated to glTF 1.0 by hand because of the manual edits to the shaders and uniforms: https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Specs/Data/Models/Box-Textured-Custom